manifests/r/RealVNC/VNCViewer/6.22.315.46706/RealVNC.VNCViewer.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: RealVNC.VNCViewer
+PackageVersion: 6.22.315.46706
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /qn REBOOT=ReallySuppress
+  SilentWithProgress: /qn REBOOT=ReallySuppress
+UpgradeBehavior: install
+ElevationRequirement: elevationRequired
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://downloads.realvnc.com/download/file/viewer.files/VNC-Viewer-6.22.315-Windows.exe
+  InstallerSha256: 788B4817E1686491924F9E3BD166265BC2B477E73A7703103F512FB02F15B5F5

```

AppsAndFeaturesEntries will be required for this app to upgrade properly
```suggestion
  InstallerSha256: 788B4817E1686491924F9E3BD166265BC2B477E73A7703103F512FB02F15B5F5
  AppsAndFeaturesEntries:
  - Publisher: RealVNC 
    DisplayName: VNC Viewer 6.22.315
    DisplayVersion: 6.22.315.46706
    InstallerType: msi
```

---

Thanks for the feedback